src/CLaserOdometry2D.cpp

```diff
@@ -86,7 +86,7 @@ void CLaserOdometry2D::init(const sensor_msgs::LaserScan& scan,
 
   //Set the initial pose
   laser_pose_    = robot_initial_pose * laser_pose_on_robot_;
-  laser_oldpose_ = laser_oldpose_;
+  laser_oldpose_ = laser_pose_;
```

Comment:
can't we just delete `laser_oldpose_`? It doesn't seem to be used anywhere?
![image](https://user-images.githubusercontent.com/60605243/120471050-84808480-c3a4-11eb-9576-4bb846139002.png)

Reply:
Done in https://github.com/Intermodalics/rf2o_laser_odometry/pull/3/commits/332acdf751419029a777717376dc33d2c44565ea.